lib/doorkeeper/oauth/refresh_token_request.rb

```diff
@@ -41,17 +43,37 @@ def default_scopes
       end
 
       def create_access_token
+        create_params = {
+          application_id:    refresh_token.application_id,
+          resource_owner_id: refresh_token.resource_owner_id,
+          scopes:            scopes.to_s,
+          expires_in:        server.access_token_expires_in,
+          use_refresh_token: true
+        }
+        create_params[:previous_refresh_token] = refresh_token.refresh_token if server.refresh_token_revoked_on_use
+
+        @access_token = AccessToken.create! create_params
+      end
+
+      def create_access_token
```

Comment:
Duplicate methods `create_access_token` at lines `45, 58` detected.<br>Assignment Branch Condition size for create_access_token is too high. [15.33/15]